aspnet/publishing/iis.rst

```diff
@@ -1,116 +1,187 @@
 Publishing to IIS
-=============================
+=================
 
-By `Rick Anderson`_ and `Luke Latham`_
+By `Rick Anderson`_ and `Luke Latham <https://github.com/GuardRex>`_
 
-.. contents:: Sections:
-  :local:
-  :depth: 1
+- `Supported Operating Systems`_
+- `IIS Configuration`_
+- `Install the Microsoft ASP.NET Core Module`_
+- `Install .NET Core on the Server`_
+- `Deploy the Application`_
+- `Configure the Website in IIS`_
+- `Create a Data Protection Registry Hive`_
+- `Common Errors`_
+- `Additional Resources`_
 
-Install the HTTP Platform Handler
------------------------------------
+Supported Operating Systems
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-- Install the HTTP Platform Handler version 1.2 or higher:
+The following operating systems are supported:
 
-  - `64 bit HTTP Platform Handler <http://go.microsoft.com/fwlink/?LinkID=690721>`_
-  - `32 bit HTTP Platform Handler <http://go.microsoft.com/fwlink/?LinkId=690722>`_
+- Windows 7 and newer
+- Windows Server 2008 R2 and newer\*
 
-If you need to enable IIS, see `IIS server configuration`_.
+\*Conceptually, the IIS configuration described in this document also applies to hosting ASP.NET Core applications on Nano Server IIS, but refer to `ASP.NET Core on Nano Server <https://docs.asp.net/en/latest/tutorials/nano-server.html>`_ for specific instructions.
 
-Configure Data Protection
------------------------------------
+IIS Configuration
```

Comment:
> I hope you'll either leave this doc as is or agree that a full tutorial is warranted. I volunteer to write it.

A full tutorial is warranted, this is just not the destination for it.

> They're looking for a fairly comprehensive end-to-end document that covers all of the basic config and that walks them through to a successfully running app.

The top-level node [Hosting](https://docs.asp.net/en/latest/hosting/index.html) is meant to be more comprehensive and I definitely see how we can shoehorn some of this great content you've written to live over there. Though not it's current shape, it should eventually read as a whitepaper for anybody hosting an ASP.NET Core application. I definitely think some of this installing IIS and troubleshooting content can live there. In fact, I would love to eventually see some more content there w.r.t. Recycling settings on AppPools, SSL Cert binding, etc., all of which are pertinent to hosting an ASP.NET Core application.

@danroth27 Do you agree? tl;dr prune this article, find a new home for some of the content